readme.md

```diff
@@ -347,7 +347,7 @@ Lifting is when you take a value and put it into an object like a [functor](#poi
 Some implementations have a function called `lift`, or `liftA2` to make it easier to run functions on functors.
 
 ```js
-const liftA2 = (f) => (a, b) => a.map(f).ap(b);
+const liftA2 = (f) => (a, b) => a.map(f).map(b);
```

Comment:
Apparently

On Sat, Aug 27, 2016, 11:32 PM hemanth.hm <notifications@github.com> wrote:

> In readme.md
> <https://github.com/hemanth/functional-programming-jargon/pull/109#discussion_r76526846>
> :
>
> > @@ -347,7 +347,7 @@ Lifting is when you take a value and put it into an object like a [functor](#poi
> >  Some implementations have a function called `lift`, or `liftA2` to make it easier to run functions on functors.
> >
> >  ```js
> > -const liftA2 = (f) => (a, b) => a.map(f).ap(b);
> > +const liftA2 = (f) => (a, b) => a.map(f).map(b);
>
> This is like the 3rd time people thought it was a typo, we need to add a
> note there?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/hemanth/functional-programming-jargon/pull/109/files/6c1dc5a09509ccd4ebdeb4aea1fb31399b735448#r76526846>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAB-4ARIjna9Q5xxeBWDV8zgyjcCH5m8ks5qkSt-gaJpZM4Ju1oJ>
> .
>